trie/sync.go

```diff
@@ -427,7 +456,10 @@ func (s *Sync) Commit(dbw ethdb.Batch) error {
 		storage int
 	)
 	for _, op := range s.membatch.nodes {
-		if op.isDelete() {
+		if !op.valid() {
+			return fmt.Errorf("invalid op, %s", op.string())
+		}
+		if op.del {
```

Comment:
Shouldn't you rewrite `isDelete` into:
```go
func (op *nodeOp) isDelete() bool {
	return op.del
}
```
Alternatively: delete the method

Reply:
Yep, the method should be deleted